packages/bsky/src/services/moderation/views.ts

```diff
@@ -155,7 +163,7 @@ export class ModerationViews {
       if (res.action === 'com.atproto.admin.defs#modEventEmail') {
         eventView.event = {
           ...eventView.event,
-          subject: res.meta?.subject ?? undefined,
+          subjectLine: res.meta?.subjectLine ?? undefined,
```

Comment:
This field is required, so perhaps we should default to `''` rather than `undefined`:
```suggestion
          subjectLine: res.meta?.subjectLine ?? '',
```